js/accordionView.js

```diff
@@ -39,11 +38,7 @@ define([
             this.model.toggleActiveItems(parseInt(itemIndex));
```

Comment:
```parseInt``` is unnecessary as itemIndex will always be an integer https://api.jquery.com/index/